Add hardening flags to winexe's executable

Signed-off-by: Fabio Fantoni <fabio.fantoni@m2r.biz>

---

--- a/source/wscript_build
+++ b/source/wscript_build
@@ -31,6 +31,8 @@
         libpath=bld.env.SAMBA_LIBS,
         rpath=bld.env.SAMBA_LIBS,
         lib=bld.env.LIBS,
+        cflags='-D_FORTIFY_SOURCE=2 -Wl,-z,relro -Wl,-z,now',
+        linkflags='-D_FORTIFY_SOURCE=2 -Wl,-z,relro -Wl,-z,now',
         use='TALLOC DCERPC',
         )
 
